roles/openshift_metrics/tasks/generate_certificates.yaml

```diff
@@ -118,13 +118,20 @@
   shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15
   register: hawkular_metrics_password
 - name: generate password for hawkular metrics jgroups
-  shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15 | base64
+  shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15
```

Comment:
We added a filter to logging to do something similiar that would not require you to 'shell'.  It might be worth as a take-away to review and see if it is worth replacing